validate/automated/nodectlCommands.md

```diff
@@ -0,0 +1,1858 @@
+---
+title: command reference guide
+hide_table_of_contents: false
+---
+
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+import MacWindow from '@site/src/components/global/MacWindow';
+
+<head>
+  <title>nodectl command reference guide</title>
+  <meta
+    name="description"
+    content="nodectl command reference guide"
+    title="Command Reference Guide"
+  />
+  <style>{`
+    :root {
+      --doc-item-container-width: 60rem;
+    }
+  `}
+  </style>
+</head>
+
+
+
+
+
+<span style={{color: "purple", fontSize:"4em", fontWeight:"800"}}>nodectl</span>
+
+## Introduction
+
+This document compliments the **nodectl** [help command](#help) reference offered through execution of a Node running the **nodectl** utility. 
+
+
+
+
+
+### What is a switch and parameter?
```

Comment:
Do we need this whole section on switch/parameters? Maybe make it one or two sentences or remove entirely. The examples in the commands section make this obvious.